project.clj

```diff
@@ -3,44 +3,47 @@
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0.html"}
 
+  :min-lein-version "2.7.0"
+
+  :parent-project {:coords [puppetlabs/clj-parent "0.1.0"]
+                   :inherit [:managed-dependencies]}
+
   ;; Abort when version ranges or version conflicts are detected in
   ;; dependencies. Also supports :warn to simply emit warnings.
   ;; requires lein 2.2.0+.
   :pedantic? :abort
 
-  :dependencies [[org.clojure/clojure "1.5.1"]
-                 ;; Logging
-                 [org.clojure/tools.logging "0.2.6"]
-                 ;; Filesystem utilities
-                 [me.raynes/fs "1.4.6"]
-                 ;; Configuration file parsing
+  :dependencies [[org.clojure/clojure]
+                 [org.clojure/tools.logging]
+                 [org.clojure/tools.cli]
+
+                 [clj-time]
+                 [me.raynes/fs]
+                 [slingshot]
+                 [cheshire]
+
                  [org.ini4j/ini4j "0.5.2"]
-                 [org.clojure/tools.cli "0.3.3"]
-                 ;; This library is used by puppetlabs.kitchensink.classpath
-                 ;; to do some classpath stuff.
                  [org.tcrawley/dynapath "0.2.3"]
                  [digest "1.4.3"]
-                 [clj-time "0.5.1"]
-                 [slingshot "0.10.3"]
-                 ;; JSON
-                 [cheshire "5.3.1"]]
+
+                 ]
 
   ;; By declaring a classifier here and a corresponding profile below we'll get an additional jar
   ;; during `lein jar` that has all the code in the test/ directory. Downstream projects can then
   ;; depend on this test jar using a :classifier in their :dependencies to reuse the test utility
   ;; code that we have.
   :classifiers [["test" :testutils]]
 
-  :profiles {:dev {:dependencies [[spyscope "0.1.4"]]
```

Comment:
No, this was some independent housekeeping.  spyscope is better off in people's `~/.lein/profiles.clj`.  I added it here years ago before I understood that.  Just noticed it now and thought it'd be a good time to go ahead and get rid of it.